sdk/cosmos/azure-cosmos-spark_3-3_2-12/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 #### Features Added
 
 #### Breaking Changes
+* Addressed `NullPointerException` in `CosmosDataItemSource` constructor when Spark runtime initialization hasn't completed yet. - See [PR 35201](https://github.com/Azure/azure-sdk-for-java/pull/35201)
```

Comment:
why is this a breaking change??

Reply:
good catch -  it should be in bug fix  section. Updated